src/test/java/org/embulk/util/json/TestJacksonFilter.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 The Embulk project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.embulk.util.json;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonPointer;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.core.filter.FilteringParserDelegate;
+import com.fasterxml.jackson.core.filter.JsonPointerBasedFilter;
+import com.fasterxml.jackson.core.json.PackageVersion;
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class TestJacksonFilter {
+    @Test
+    public void testFilterSequenceSkippingNonMatch() throws IOException {
+        final com.fasterxml.jackson.core.JsonParser parser = createFilteredParser(
```

Comment:
What do you think to add this comment ?
Because, I completely understand the purpose of this test after reading the google group article.

```java
// This |parser| returns {"bar":"baz"}, and {"bar":"qux"}, with just skipping {"xxx":{"yyy":"zzz"}}.
```

Reply:
Thanks. Sounds good! Added.